gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -237,10 +270,25 @@ Dump::visit (BlockExpr &block_expr)
 {
   stream << "BlockExpr: [";
   indentation.increment ();
-  stream << std::endl;
+  stream << "\n";
+
   // TODO: inner attributes
+  stream << indentation;
```

Comment:
At least for me, it seems more readable to put those on one line to better correspond to the output format.

Reply:
Sure, I fixed it :)